### ISSUE TITLE
Sync Wi-Fi settings

### ISSUE THREAD

**Issue by PureTryOut** [AsteroidOS/AsteroidOSSync#126]
Today I found out my watch supports connecting to Wi-Fi. I can do this via the command-line, but I was thinking it would be great if it could sync the settings from the phone that's already connected to a local network. This probably shouldn't happen automatically but have the user choose if the watch should connect or not.

**Comment by jrtberlin** [comment]
That's a good idea. We can get the SSID list and let the user enter the password on the phone.

**Comment by PureTryOut** [comment]
Yup! I don't know if it's possible but it would be nice if it could take a stored password as well so you don't get prompted when you have already connected to the network with your phone.

**Comment by jrtberlin** [comment]
There is a system API for that and wpa supplicant conf files, but using/reading them is only possible as a privledged application.
That said we could offer that on rooted devices as an alternative.

**Comment by PureTryOut** [comment]
Ah ok too bad. Well in that case asking to enter the password on the phone is fine too. And yes `if rooted` would be nice.
